assets/scripts/build-reference-pages.js

```diff
@@ -0,0 +1,1170 @@
+#!/usr/bin/env node
+const lodash = require('lodash');
+const yaml = require('js-yaml');
+const fs = require('fs');
+const marked = require('marked');
+const slugify = require('slugify');
+const $RefParser = require('@apidevtools/json-schema-ref-parser');
+const safeJsonStringify = require('safe-json-stringify');
+
+const supportedLangs = ['en'];
+
+/**
+ * Update the menu yaml file with api
+ * @param {object} apiYaml - object with data
+ * @param {string} apiVersion - string with version e.g v1
+ */
+const updateMenu = (specData, specs, languages) => {
+
+  languages.forEach((language) => {
+    const currentMenuYaml = yaml.safeLoad(fs.readFileSync(`./config/_default/menus/menus.${language}.yaml`, 'utf8'));
+
+  // filter out auto generated menu items so we just have hardcoded ones
+  const newMenuArray = (currentMenuYaml[`api`] || []).filter((entry => !entry.hasOwnProperty("generated")));
+
+  specData.forEach((apiYaml, index) => {
+    const apiVersion = specs[index].split('/')[3];
+    // now add back in all the auto generated menu items from specs
+    apiYaml.tags.forEach((tag) => {
+
+      const tagSlug = getTagSlug(tag.name);
+      const existingMenuItemIndex = newMenuArray.findIndex((i) => i.identifier === tagSlug);
+      if(existingMenuItemIndex > -1) {
+        // already exists
+        // newMenuArray[existingMenuItemIndex].params["versions"].push()
+      } else {
+        // doesn't exist lets add it
+        newMenuArray.push({
+          name: tag.name,
+          url: (language === 'en' ? `/api/latest/${tagSlug}/` : `/${language}/api/latest/${tagSlug}/` ),
+          identifier: tagSlug,
+          generated: true
+        });
+      }
```

Comment:
Was line 34 intentionally left commented out, or there for future use? If not, then maybe we remove the if/else?

Reply:
its more specific to the api schema parsing than ref docs. Thanks for pointing it out i'll do some more cleanup in another PR